src/poker_now_log_converter/game.py

```diff
@@ -106,7 +107,69 @@ def __init__(self, poker_now_log: List[List[str]], currency: str = "USD", timezo
                     current_hand.get_seat_by_player_name_with_id(
                         current_hand.dealer.player_name_with_id).seat_desc += " (button)"
 
-            elif "raises" in line:
+            elif "posts a bet of" in line and "bomb pot bet" in line:
```

Comment:
Should this line have the included double quote at the start as well?

Reply:
I suppose they could, but since these are multi-word phrases, you'd essentially need someone TRYING to screw with the system for those to be misinterpreted with actions. I think it's really dealer's choice on whether they want to include the escape or not.